DESCRIPTION

```diff
@@ -9,15 +9,18 @@ Authors@R: c(person("Bioconductor Package Maintainer", role = c("aut", "cre"),
 		     email = "maintainer@bioconductor.org"),
 	      person("Valerie", "Obenchain", role = "aut"),
 	      person("Michael", "Love", role = "aut"),
-	      person("Lori", "Shepherd", role = "aut"),
-	      person("Martin", "Morgan", role = "aut"))
+	      person("Martin", "Morgan", role = "aut"),
+              person("Sonali", "Kumari", role = "ctb",
+                     comment = "Converted 'GenomicFiles' vignettes from Sweave to RMarkdown / HTML."),
+              person("Bioconductor Package Maintainer",
+                     email = "maintainer@bioconductor.org", role = "cre"))
```

Comment:
I think these lines are duplicating the information in 8-9. Let's remove these lines, but remember to replace the comma on line 14 with a `)`.

Reply:
Done